cli/internal/util/env/env_test.go

```diff
@@ -0,0 +1,147 @@
+package env
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestGetHashableEnvPairs_SingleVar(t *testing.T) {
```

Comment:
This is a perfect situation for the table test pattern. In VSCode if you right-click on `GetConfigEnvPairs` and tell it to `Generate Unit Tests For Function` it will set up that structure for you.

With a bit of renaming it will make it far easier to identify the actual thing you're attempting to test.